app/views/adoption_application_mailer/_adoption_application_mailer_content.html.erb

```diff
@@ -0,0 +1,50 @@
+<table style="border-collapse: collapse;">
+  <%= render "table_row", adoption_application: adoption_application,
```

Comment:
@halogenandtoast This suggestion is exactly what I hoped was possible but couldn't see how to do. I've changed it slightly but thank you so much for the guidance.